instrumentation/otel-webserver-module/src/nginx/ngx_http_opentelemetry_module.c

```diff
@@ -965,26 +1003,36 @@ static ngx_flag_t ngx_initialize_opentelemetry(ngx_http_request_t *r)
     return false;
 }
 
-static void stopMonitoringRequest(ngx_http_request_t* r)
+static void stopMonitoringRequest(ngx_http_request_t* r,
+    const char* request_handle_key)
 {
-    if(r->internal)
-    {
-        return;
```

Comment:
Why do we need to remove this condition?

Reply:
StopMonitoringRequest would be called from log handler, which comes only once in the lifetime of the request and r->internal can be either 1 or 0. We process only when request handle key is valid.